doxygen/lang/900_release_notes.dox.tmpl

```diff
@@ -192,6 +192,9 @@
       - <a href="../../modules/Swagger/html/index.html">Swagger</a>:
         - Swagger module does not accept multipart/form-data content-type and also does not work with list and hash parameters
           (<a href="https://github.com/qorelanguage/qore/issues/2932">issue 2932</a>)
+      - <a href="../../modules/WebSocketHandler/html/index.html">WebSocketHandler</a>:
+        - added stopping connection from server side via @ref WebSocketHandler::WebSocketConnection::stop(), @ref WebSocketHandler::WebSocketHandler::stopOne()
```

Comment:
links from qore to user modules do not yet work - I'm not sure how we can solve that without building the documentation twice - the problem is that the tag files for referenced documentation need to be present when the docs are built